### PR TITLE
beanstalkd: add page

### PR DESCRIPTION
**Pull request by sbrl**
I saw this program a while back and thought it had such a cool name! I don't have a use for it yet, but by adding it to tldr-pages I'm not as likely to forget about it :P

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/beanstalkd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Start beanstalkd listening on a custom port and address:
+
+`beanstalkd -l {{ip_address}} -p {{port_number}}`
+
+- Persist work queues by saving them to disk:
+
+`beanstalkd -b {{path/to/persistence/directory}}`
+
+- Sync to the persistence directory every X milliseconds:
```
Comment (agnivade) [inline pages/common/beanstalkd.md]: Maybe just put a number like 500. Looks a bit better to me :smile:

File: pages/common/beanstalkd.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+> A simple and generic work-queue server.
+
+- Start beanstalkd, listening on port 11300:
+
+`beanstalkd`
+
+- Start beanstalkd listening on a custom port and address:
+
+`beanstalkd -l {{ip_address}} -p {{port_number}}`
```
Comment (agnivade) [inline pages/common/beanstalkd.md]: Specifying a different ip address is only possible when a machine has 2 ip addresses right ? Or did you mean like `localhost` and the public interface ?

Comment (sbrl) [reply pages/common/beanstalkd.md]: I meant to say the IP that it listens on. It defaults to `0.0.0.0`.

Comment (agnivade) [reply pages/common/beanstalkd.md]: Hmm .. ok.

**Pull request merged**